### PR TITLE
[CFM-1388] purl for group content

### PR DESCRIPTION
**Pull request by nirgn975**
https://issuetracker.amplexor.com/jira/browse/CFM-1388

The og repo: https://github.com/capacity4dev/og.git

### CODE REVIEW

**Comment by nirgn975** [comment]
@ordavidil Ready for code review (read the issue comments at JIRA).

File: config/make/og.make [hunk 1]
```diff
@@ -3,10 +3,7 @@ api = 2
 
 ; Organic Groups and friends.
 
-projects[og][download][type] = "git"
-projects[og][download][url] = "https://github.com/capacity4dev/og.git"
-projects[og][download][branch] = og_context_check_access
-projects[og][type] = "module"
+projects[mailsystem][version] = "7.x-2.9"
```
Comment (ordavidil) [inline config/make/og.make]: @nirgn975 
the `mailsystem` does not seems related over here.

**Comment by ordavidil** [comment]
@nirgn975 
please re-roll

**Comment by ordavidil** [comment]
@amitaibu 
After changing the make file to download _og_ from its original source, we get some access issues.

Seeking the code, I've found that `og_context()` is being used within _access callback_ of a menu item, but it returns `FALSE` instead of the context.

When it gets to the _page callback_ then the _og_context()` works just fine.

**Comment by nirgn975** [comment]
Managed to find the problem:
page_not_found___capacity4more

When you are in a `photoalbum`, click the image an then on the `Back to photo album`. For example:
lusail_soccer_stadium___capacity4more

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/theme/node-gallery-api-item-navigator.tpl.php [hunk 2]
```diff
@@ -23,7 +23,13 @@
   <div class="navigator--center">
     <?php if ($gallery_link): ?>
       <div class="back-link">
-        <?php print l(t("Back to photo album"), $gallery_link) ?>
+        <?php
+          global $base_path;
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/theme/node-gallery-api-item-navigator.tpl.php]: @nirgn975 
Let's move this logic to a preprocess function...

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/c4m_content_photo.module [hunk 3]
```diff
@@ -55,3 +55,26 @@ function c4m_content_photo_c4m_helper_entity_label_info() {
...
+  if ($hook != 'node_gallery_api_item_navigator') {
+    return;
+  }
+
+  global $base_path;
+  $options = array(
+    'absolute' => TRUE,
+  );
+
+  // 'gallery_link' contain the full path to the gallery (back button). This create duplication of the base path in
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/c4m_content_photo.module]: @nirgn975 
Is it just me this one is gonna fail on the sniffer for a line that is longer than 80 chars ?
:\

**Comment by ordavidil** [comment]
Merged under #1102

**Pull request closed without merging**